key.ui/src/main/java/de/uka/ilkd/key/util/CommandLine.java

```diff
@@ -189,6 +185,42 @@ protected void print(PrintStream ps, int descriptionCol) {
     public CommandLine() {
     }
 
+    private String usedSubCommand = "";
+
+    private final Map<String, CommandLine> subcommands = new HashMap<>();
+
+    public CommandLine addSubCommand(String name) {
+        if (name.startsWith(MINUS)) {
+            throw new IllegalArgumentException(
+                "Subcommands must not start with '" + MINUS + "': " + name);
+        }
+        if (subcommands.containsKey(name)) {
+            throw new IllegalArgumentException(name + " has already been registered");
+        }
+        CommandLine subCommand = new CommandLine();
+        subcommands.put(name, subCommand);
+        return subCommand;
+    }
+
+    public CommandLine getSubCommandLine(String name) {
```

Comment:
`getSubCommandLine` → `getSubCommand`

Reply:
I changed the name as suggested and added documentation.